fog/types/src/common.rs

```diff
@@ -87,4 +113,41 @@ mod tests {
         assert!(!range.overlaps(&BlockRange::new(0, 10)));
         assert!(!range.overlaps(&BlockRange::new(13, 100)));
     }
+
+    #[test]
+    fn from_string_well_formatted_creates_block_range() {
+        let start_block = 0;
+        let end_block = 10;
+        let block_range_str = format!("{},{}", start_block, end_block);
```

Comment:
I think if desired it could be a follow on PR/Issue, but `3, 5` will fail to parse.  My hunch is hat most users will want to put a space after the comma

Reply:
This probably won't be an issue because these ranges will be passed as CLI args (and CLI convention is to not use whitespace between arguments). But I implemented it and added a test.